src/SwapExtractionTool/Program.cs

```diff
@@ -22,7 +22,7 @@ static async Task Main(string[] args)
             }
             else
             {
-                startBlock = 1_949_800;
+                startBlock = 1975500;
```

Comment:
@StratisIain Is it ok to update the start block here to something closer to when we started the swap?